modules/light-clients/08-wasm/types/export_test.go

```diff
@@ -8,3 +8,10 @@ package types
 type InstantiateMessage struct {
 	instantiateMessage
 }
+
+type (
+	// queryMsg is used to encode messages that are sent to the contract's query entry point.
+	QueryMsg = queryMsg
```

Comment:
would also need em for each individual payload, right?

Reply:
i think this pr is superceded by #4879, as per discussion on the related issue/in the planning meeting this morning, i will close this one.